### DISCUSSION TITLE
The GitHub Enterprise Server 3.13 release is generally available

### DISCUSSION THREAD

**Discussion by goldieowner17** [community/community#128980] [Enterprise]
![GHES-Header](https://github.com/community/community/assets/166861992/fbf1397e-d459-438e-9b7f-fd1823feb32f)

### GitHub Enterprise Server 3.13 gives customers more fine-grained control over deployment requirements and enhanced security controls. 

Here are a few highlights:

- We are introducing a new feature for repositories called custom properties, a major enhancement to how repositories are managed and classified across GitHub organizations. Properties offer a flexible way to add meaningful metadata to your repositories that simplifies repository classification, enhances discoverability, and seamlessly integrates with rulesets. Check out the [demo](https://www.youtube.com/watch?v=z0CYdcqZxyQ)! For more information, see [custom properties for repositories](https://docs.github.com/en/organizations/managing-organization-settings/managing-custom-properties-for-repositories-in-your-organization).
- Elasticsearch will be upgraded from version 5 to version 8, when the appliance is upgraded to 3.13. Elasticsearch powers all search experiences in GHES including code search and audit logs. Upgrading ES5 to ES8 allows the platform to take advantage of better performance and improved security posture in ES8. For more information regarding what to expect during ES8 upgrade, see [Preparing for Elasticsearch upgrade in GHES 3.13](https://docs.github.com/en/enterprise-server@3.13/admin/monitoring-managing-and-updating-your-instance/updating-the-virtual-machine-and-physical-resources/preparing-for-the-elasticsearch-upgrade).
- Enterprise and organization audit log events now include the applicable SAML and SCIM identity data associated with the user. For more information, see [Reviewing the audit log for your organization](https://docs.github.com/en/enterprise-server@3.13/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization).
- Developers who use devcontainer.json files to define their development containers will now be able to use Dependabot version updates to keep their dependencies in the container up-to-date. Once configured in dependabot.yml, Dependabot will open PRs on a specified schedule to update the listed dependencies to latest.
- Pull Requests rebases are now faster! Under the hood, rebase commits now use the [merge-ort](https://github.blog/changelog/2023-06-28-rebase-commits-now-created-using-the-merge-ort-strategy). Rebases that timed out for large repositories before are now a lot more likely to be successful.
- Using Project Status Updates, you can now provide high level details on the status, timing, and progress of your project, directly from the project! This makes it easy to know and share with others how your work is progressing, any risks, and a history of when and why something changed, all in the same place where you’re tracking your work.

**Comment by rauanmayemir** [comment]
Is there an upgrade path from 3.13RC1 to 3.13 GA? I'm looking for a way to switch the qcow2 image.

**Reply by pallsama** [reply]
Thanks for asking.  There is no upgrade path from 3.13 RC1 to GA.

**Reply by rauanmayemir** [reply]
How do you mean, do I just replace the qcow image?
